resources/qml/Preferences/MachinesPage.qml

```diff
@@ -136,6 +136,11 @@ UM.ManagementPage
         {
             id: confirmDialog
             object: base.currentItem && base.currentItem.name ? base.currentItem.name : ""
+            text: base.currentItem && base.currentItem.hasCloudConnection?
```

Comment:
This does mean that _all_ cloud printers will have this behavior, which is not quite correct. If someone else makes a plugin that adds a cloud printer in the old way, this message will then also show up (which is not correct, since the printer will not be re-added)